README.md

```diff
@@ -95,17 +129,39 @@ rspec ./spec --format documentation
 
 [GitHub](https://github.com/blase147) | [Twitter](https://twitter.com/ChukwumaMosanya) | [LinkedIn](https://www.linkedin.com/in/chukwuma-mosanya)
 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Views.
+- Forms
+- Validations, Model specs, and n+1 problems.
+- Add Devise.
+- Add authorization rules.
+- Integration specs for Views.
+- Add API endpoints.
+- API documentation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/blase147/creating_data_model/issues/new).
+Feel free to check the [issues page](https://github.com/blase147/firmtecs_blog/issues/new).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-Give this project a star if you like it !
+> Give this project a star if you like it...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
```

Comment:
- [ ] Kindly fix the broken broken link to the MIT license agreement as requested by the previous reviewer
![image](https://user-images.githubusercontent.com/81932478/217265425-f7c4e89b-0ce9-4462-abdd-f7c49a1190d9.png)